csrc/python_frontend/fusion_definition.cpp

```diff
@@ -133,11 +134,15 @@ void FusionDefinition::finalizeSchedule(
   FusionGuard::setCurFusion(prev_fusion_);
   prev_fusion_ = nullptr;
 
-  user_sched_->executor->compileFusion(
-      user_sched_->schedule.get(),
-      inputs,
-      user_sched_->fusion_id_,
-      user_sched_->device_id_);
+#ifdef NVFUSER_DISTRIBUTED
```

Comment:
Is this line needed? Are anything in the block only declared under NVFUSER_DISTRIBUTED?

Reply:
`multidevice_flag` and `multi_device_executor` is hidden behind the flag.

If i'm to refactor `multi_device_executor` to replace `multidevice_flag`, I'll need this macro I think.